projects/schematics/package.json

```diff
@@ -46,5 +46,5 @@
       "@spartacus/incubator"
     ]
   },
-  "main": "./public_api.js"
+  "main": "src/public_api.js"
```

Comment:
This has to use `.js`?

Reply:
Yes, because when app is built and bundled, it's operating only on .JS files. So we're pointing to already transpiled/bundled file here.